src/pages/collections/[id].tsx

```diff
@@ -0,0 +1,34 @@
+import { useContext, useEffect } from "react";
+import Layout from "@/components/layout/layout";
+import { Heading } from "@nypl/design-system-react-components";
+import { breadcrumbsContext } from "../breadcrumbsContext";
+
+const Collection = () => {
+  const { breadcrumbs, setBreadcrumbs } = useContext(breadcrumbsContext);
+  useEffect(() => {
+    setBreadcrumbs({ text: "A Collection", url: "/collections/collection" });
+  }, []);
+  return (
+    <Layout activePage="collection" breadcrumbs={breadcrumbs}>
+      <Heading>
+        <>Collection</>
+      </Heading>
+    </Layout>
+  );
+};
+
+// Imagine an API call somewhere in here
```

Comment:
Have you tried passing this mocked API-fetched data to the component? How would this change if you use this `breadcrumbs` array as a prop in `Collection`, i.e. `const Collection = ({ breadcrumbs }) => {....`.

You could use the last object in the array as the value that is passed to `setBreadcrumbs`.

Reply:
Right, sorry for the confusion of trying to represent two approaches within the same file but yes imagining that as a prop to `Collections`. If that's the case though, I wouldn't use the context at all